### ISSUE TITLE
is the update so being worked on?

### ISSUE THREAD

**Issue by HoodBlah** [skyjay1/Nomadic-Tents#86]
New issues seemed to have stopped and no news of an update is around so I just wanted to ask if a 1.20.1 version is still in the works. This is a one of a kind mod and I would hate for it to be left in the books of history as another great mod left in dust.

**Comment by KiriCattus** [comment]
Yes, I'm nearly done, just a few hours of cleanup and it'll be ready
